### PR TITLE
pulumi: add page

### PR DESCRIPTION
**Pull request by Codecritics**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by Codecritics** [comment]
Hello everyone,
I opened this PR because of this request: https://github.com/tldr-pages/tldr/issues/8183.
Thanks for everything.

**Comment by CleanMachine1** [comment]
closes #8183

File: pages/common/pulumi.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+- Create a new project using a template:
+
+`pulumi new`
+
+- Create a new stack, an isolated deployment target:
+
+`pulumi stack init`
+
+- Configure variables such as keys, regions, and so on:
```
Comment (CleanMachine1) [inline pages/common/pulumi.md:14]: Does this open a interactive command or is there other parameters that need to go with it?

Comment (Codecritics) [reply pages/common/pulumi.md:14]: It opens an Interactive command  but you can also provide additional parameters.

Comment (CleanMachine1) [reply pages/common/pulumi.md:14]: such as?

File: pages/common/pulumi.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+`pulumi config`
+
+- Preview and deploy changes to your program and/or infrastructure:
```
Comment (CleanMachine1) [inline pages/common/pulumi.md:18]:
```suggestion
- Preview and deploy changes to a program and/or infrastructure:
```

We avoid using `you` and `your`

File: pages/common/pulumi.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+`pulumi up`
+
+- Preview your changes explicitly before deploying:
```
Comment (CleanMachine1) [inline pages/common/pulumi.md:22]:
```suggestion
- Preview changes explicitly before deploying:
```

File: pages/common/pulumi.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+`pulumi preview`
+
+- Destroy your program and its infrastructure when you’re done:
```
Comment (CleanMachine1) [inline pages/common/pulumi.md:26]:
```suggestion
- Destroy a program and its infrastructure after execution:
```
```suggestion
- Destroy a program and its infrastructure:
```

**Review by CleanMachine1 — CHANGES_REQUESTED** [review]

File: pages/common/pulumi.md (hunk 1 above)
Comment (Codecritics) [reply pages/common/pulumi.md:14]: Such as an Organization name or/and a Stack Name

Comment (CleanMachine1) [reply pages/common/pulumi.md:14]: Do you input them once when the command is running or as parameters in the command?

Comment (Codecritics) [reply pages/common/pulumi.md:14]: You input them when the command is running

Comment (CleanMachine1) [reply pages/common/pulumi.md:14]: Ah, ok thanks.


```suggestion
- Interactively configure variables such as keys, regions, etc: 
```

**Review by CleanMachine1 — APPROVED** [review]

File: pages/common/pulumi.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
+# pulumi
+
+> Define infrastructure on any cloud using familiar programming languages.
+> More information: <https://www.pulumi.com/docs/>.
```
Comment (CleanMachine1) [inline pages/common/pulumi.md:4]:
```suggestion
> More information: <https://www.pulumi.com/docs/reference/cli>.
```

File: pages/common/pulumi.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
+# pulumi
+
+> Define infrastructure on any cloud using familiar programming languages.
+> More information: <https://www.pulumi.com/docs/reference/cli>.
+
+- Create a new project using a template:
+
+`pulumi new`
+
+- Create a new stack, an isolated deployment target:
```
Comment (navarroaxel) [inline pages/common/pulumi.md:10]: I think a connector is missing here `with`? `using`? `creating`?
```suggestion
- Create a new stack using an isolated deployment target:
```

File: pages/common/pulumi.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
+`pulumi stack init`
+
+- Interactively configure variables such as keys, regions, etc:
```
Comment (navarroaxel) [inline pages/common/pulumi.md:14]: I prefer start with the action verb, it's clear
```suggestion
- Configure variables (e.g. keys, regions, etc.) interactively:
```

File: pages/common/pulumi.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+
+`pulumi config`
+
+- Preview and deploy changes to a program and/or infrastructure:
```
Comment (navarroaxel) [inline pages/common/pulumi.md:18]: This command has a confirmation prompt?

Comment (Codecritics) [reply pages/common/pulumi.md:18]: That command lists all configuration values for a specific stack. 
To add a new configuration value  `pulumi config set`. 
To remove an existing value run `pulumi config rm`. 
To get the value of for a specific configuration key, use `pulumi config get`

**Review by kbdharun — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**